docs/source/admin/fs/rest.rst

```diff
@@ -251,6 +251,18 @@ The field ``external`` doesn't necessarily be a flat structure. This is a more a
 
 .. attention:: Only standard :ref:`FSCrawler fields <generated_fields>` can be set outside ``external`` field name.
 
+
+Specifying an elasticsearch index
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+By default, fscrawler creates document in the index defined in the _settings.yaml (or json) file.
```

Comment:
Could you add a blank line just before this paragraph?

Reply:
Could you change the sentence from `defined in the _settings.yaml (or json) file.` to `defined in the ``_settings.yaml`` file.`.